metadata_backend/conf/schemas.json

```
@@ -124,5 +124,14 @@
             "url": "https://github.com/imi-bigpicture/metadata-schema/blob/v0.1.1/src/BP.sample.xsd"
         },
         "provider": "BP"
+    },
+    "observation": {
```

```suggestion
    "bpobservation": {
```

let us keep all with the bp prefix just in case

---

Ok, then `image` should probably be changed to `bpimage` as well?

---

yes